pkg/agent/openflow/feature_templates.go

```diff
@@ -0,0 +1,214 @@
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package openflow
+
+import (
+	binding "antrea.io/antrea/pkg/ovs/openflow"
+	"antrea.io/antrea/pkg/util/runtime"
+)
+
+func (c *featurePodConnectivity) getTemplate(protocol ofProtocol) *pipelineTemplate {
+	var template *pipelineTemplate
+	if protocol == ofProtocolIP {
+		template = &pipelineTemplate{
+			stageTables: map[binding.StageID][]tableRequest{
+				binding.ClassifierStage: {
+					tableRequest{ClassifierTable, 0x7f},
```

Comment:
anywhere we can learn why here is setting as `0x7f`? or will there be any update for existing ovs-pipeline design doc? https://github.com/antrea-io/antrea/blob/main/docs/design/ovs-pipeline.md

Reply:
I agree. How the table priorities are decided? Should we define constants?

Reply:
The order of table within a stage is decided by this priority. We don't need to define constants because priorities should be the same within a stage, otherwise the order of tables cannot be decided by priority. @jianjuns @luolanzone

Reply:
I do not understand your explanation. How 0x7f, 0x6f, 0x7d in this file are decided? If I add a new table later, how I decide the priority value?